App/Containers/TextilePhotos.js

```diff
@@ -17,13 +19,14 @@ class TextilePhotos extends React.PureComponent {
         <Image style={navStyles.headerTitleImage} source={require('../Images/TextileHeader.png')} />
       </TouchableWithoutFeedback>
     ) : params.thread
+
     return {
       headerTitle
     }
   }
 
   componentDidMount () {
-    this.props.navigation.setParams({ 
+    this.props.navigation.setParams({
```

Comment:
Anything actually change in this file? Looks like some added imports that aren't used, etc?

Reply:
you might have been reviewing prior to my last commit, i had been messing around with it to gen an invite link for testing... should be reverted now